processor_function/shared/cnab_builder.py

```diff
@@ -64,17 +83,17 @@ def _get_network_profile(self) -> ContainerGroupNetworkProfile:
             raise Exception("No network profile")
 
         network_profile = ContainerGroupNetworkProfile(id=net_result.id)
+
         return network_profile
 
     def _setup_aci_deployment(self) -> ContainerGroup:
```

Comment:
This function/methods needs a description.

Reply:
removed the function as the whole network thing went out